components/typography/styles.js

```diff
@@ -506,10 +506,12 @@ export const baseTypographyStyles = css`
 	html {
 		--d2l-document-direction: ltr;
 		--d2l-mirror-transform: none;
+		--d2l-mirror-end: right;
```

Comment:
Having "mirror" in the name here is making me expect `end` to be flipped to `left`. Does `--d2l-end` make sense?

Reply:
Sounds good to me.